# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../xrandr/compile.patch
# Copyright (C) 2007 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

8.0-trunk-arm-gnueabi-arm920t-cross/usr/include/bits/sched.h:75: error: previous declaration of 'clone' was here

  - Rene Rebe <rene@exactcode.de>

--- xrandr-1.2.2/xrandr.c	2007-07-05 05:03:34.000000000 +0200
+++ xrandr-1.2.2-fixed/xrandr.c	2007-08-12 15:53:10.000000000 +0200
@@ -189,7 +189,7 @@
 
 #if HAS_RANDR_1_2
 typedef enum _policy {
-    clone, extend
+    doclone, extend
 } policy_t;
 
 typedef enum _relation {
@@ -1602,7 +1602,7 @@
     int		ret = 0;
 #if HAS_RANDR_1_2
     output_t	*output = NULL;
-    policy_t	policy = clone;
+    policy_t	policy = doclone;
     Bool    	setit_1_2 = False;
     Bool    	query_1_2 = False;
     Bool	modeit = False;
@@ -1863,7 +1863,7 @@
 	    continue;
 	}
 	if (!strcmp ("--clone", argv[i])) {
-	    policy = clone;
+	    policy = doclone;
 	    setit_1_2 = True;
 	    continue;
 	}
@@ -2296,9 +2296,9 @@
 		printf ("\tClones:    ");
 		for (j = 0; j < output_info->nclone; j++)
 		{
-		    output_t	*clone = find_output_by_xid (output_info->clones[j]);
+		    output_t	*doclone = find_output_by_xid (output_info->clones[j]);
 
-		    if (clone) printf (" %s", clone->output.string);
+		    if (doclone) printf (" %s", doclone->output.string);
 		}
 		printf ("\n");
 		if (output->crtc_info)
